.idea/laravel-event-sourcing.iml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove the `.idea` directory from this PR.

It might be helpful to create a global gitgnore to avoid future problems.